src/enterprise-edge/azext_enterprise_edge/_params.py

```diff
@@ -17,3 +17,6 @@ def load_arguments(self, _):
     with self.argument_context('staticwebapp enterprise-edge') as c:
         c.argument("name", arg_type=staticsite_name_arg_type)
         c.argument("resource_group_name", arg_type=resource_group_name_type)
+
+    with self.argument_context('staticwebapp enterprise-edge') as c:
```

Comment:
Shouldn't the default=True for them to be able to register? Add the fwd link from above here instead

Reply:
I thought we wanted it to register the `Microsoft.CDN` provider by default, so hence I made the default for `--no-register` `False`